### PR TITLE
Add templating of turing ingress for networking api v1beta1 and v1.

### PR DESCRIPTION
**Pull request by ashwinath**
Some clusters may be on a lower version of Kubernetes (< 1.19) and do not allow for the Ingress object under the `networking.k8s.io/v1` API but allow for `networking.k8s.io/v1beta1` API. This PR allows for either configuration.

### CODE REVIEW

File: infra/charts/turing/templates/ingress.yaml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 {{- if .Values.turing.ingress.enabled }}
-apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: {{ template "turing.fullname" . }}
   namespace: {{ .Release.Namespace }}
   annotations:
-    kubernetes.io/ingress.class: {{ .Values.turing.ingress.class }}
+    kubernetes.io/ingress.class: "{{ .Values.turing.ingress.class }}"
```
Comment (ashwinath) [inline infra/charts/turing/templates/ingress.yaml:7]: IIRC empty string is valid as well but not nil object.

**Comment by ashwinath** [comment]
I tested the template generation of the ingress with `turing.ingress.useV1Beta1 = true` in a Kubernetes 1.18 environment.

File: infra/charts/turing/Chart.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: "Turing: ML Experimentation System"
 name: turing
-version: 0.2.0
+version: 0.2.1
```
Comment (romanwozniak) [inline infra/charts/turing/Chart.yaml:4]: This should be `0.2.2` now

**Review by romanwozniak — APPROVED** [review]
The same thing should probably be done for mlp/merlin ingresses as well, to keep it consistent. Besides this, LGTM

File: infra/charts/turing/Chart.yaml (hunk 2 above)
Comment (ashwinath) [reply infra/charts/turing/Chart.yaml:4]: ok I'll bump it up.

**Pull request merged**